setup.py

```diff
@@ -53,7 +53,10 @@
     "jsonschema~=3.2.0",
     "importlib_metadata;python_version<'3.8'",
     "azure-iot-device~=2.11",
+    "tomli",
+    "tomli-w",
     "tqdm~=4.62",
+    "treelib",
```

Comment:
would like to see a pipeline run with 3.7 to make sure everything still works

Reply:
https://dev.azure.com/azureiotdevxp/aziotcli/_build/results?buildId=7095&view=logs&j=6b37a07b-ae07-5f7f-46c3-5ce679ba41f2&t=6495c3e0-3a6f-5c64-f017-67dfb92d7cfa